### DISCUSSION TITLE
securityonion-2.3.52 install error

### DISCUSSION THREAD

**Discussion by osotob** [Security-Onion-Solutions/securityonion#4707] [Unsupported Versions]
Hi
I can't install the SO-2.3.52 from iso. Always stops at this point:

Command failed with exit code 1; will retry in 10 seconds (49 / 50)...
Executing command with retry support: docker pull ghcr.io/security-onion-solutions/so-acng:2.3.52
Error response from daemon: Get https://ghcr.io/v2/: proxyconnect tcp: tls: first record does not look like a TLS handsh                                  ake
Results:  (1)
Command failed with exit code 1; will retry in 10 seconds (50 / 50)...
Command continues to fail; giving up.
Executing command with retry support: curl -A 'netinstall/2.3.52/centos/3.10.0-1160.25.1.el7.x86_64' https://sigs.securi                                  tyonion.net/2.3.52/so-acng:2.3.52.sig --output /root/sosigs/so-acng:2.3.52.sig
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   543  100   543    0     0    640      0 --:--:-- --:--:-- --:--:--   641
Results:  (0)
There is a problem downloading the so-acng:2.3.52 image. Details:

gpg: Signature made Thu 03 Jun 2021 02:25:33 PM UTC using RSA key ID FE507013 gpg: BAD signature from "Security Onion So                                  lutions, LLC <info@securityonionsolutions.com>"
Errors detected during setup; skipping post-setup steps to allow for analysis of failures.
2021-07-06T10:36:02Z | I | Removing so-setup permission entry from sudoers file

Any help please?
Thanks

**Comment by weslambert** [comment]
Could there be a proxy/firewall causing issues?

**Reply by osotob** [reply]
Hi weslambert
Yes it was. I could solve this issue by changing https:// to http:// in the config.
Thanks
